src/draganddrophandler.js

```diff
@@ -4,27 +4,39 @@ export const dragStart = (event) => {
   const taskItemContainer = event.target.closest('.list-item');
   event.dataTransfer.setData('text/plain', taskItemContainer.dataset.index);
   taskItemContainer.classList.add('dragging');
+  console.log("dragStart"+ taskItemContainer.dataset.index);
```

Comment:
- [ ] In accordance to best practises, please do not commit `console.log` to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. **Note: This was not there in the previous review, it would had been pointed out. ℹ️**